src/java/picard/analysis/CollectWgsMetricsFromSampledSites.java

```diff
@@ -0,0 +1,44 @@
+package picard.analysis;
+
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.util.*;
+import picard.cmdline.CommandLineProgramProperties;
+import picard.cmdline.Option;
+import picard.cmdline.programgroups.Metrics;
+
+import java.io.File;
+
+/**
+ * Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing experiments,
+ * but only at a set of sampled positions.
```

Comment:
It assumes that the positions are samples in such a way so that every record may have *at most* one sampled position. this needs to be made clear if this is to be useful for others.